extensions/amp-ad/0.1/test/test-amp-ad.js

```diff
@@ -107,7 +107,9 @@ function tests(name, installer) {
           type: 'a9',
           src: 'testsrc',
           resizable: '',
-        }, 'https://schema.org').then(element => {
+        }, 'https://schema.org', undefined, () => {
+          clock = sandbox.useFakeTimers();
```

Comment:
this actually made us find the ad bug that @jridgewell found too. the reason why this was not an issue previously was that each test used to invoke `installAd(win)` which would have a clean/reset `loadingAdsCount`. we could also just add a `resetAdCtrlForTesting` if that is acceptable